runbotics/runbotics-orchestrator-ui/src/views/process/AddProcess.tsx

```diff
@@ -34,6 +36,14 @@ const formData: IProcess = {
     definition: emptyBpmn,
 };
 
+enum ErrorType {
+    NAME_NOT_AVAILABLE,
+}
+
+const errorMessages: Record<ErrorType, string> = {
+    [ErrorType.NAME_NOT_AVAILABLE]: translate('Process.Error.NameNotAvailable'),
```

Comment:
as mentioned above `ErrorType.NAME_NOT_AVAILABLE` returns numeric value which is not correct for js objects (you cannot do it directly if you create new object